.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "parserOptions": {
+      "ecmaVersion": 2017
+    },
+    "env": {
+      "browser": true,
+      "es6": true
+    },
+    "extends": ["eslint:recommended", "google"],
+    "rules": {
+      "no-multi-spaces": "off",
+      "require-jsdoc": "off",
+      "valid-jsdoc": "off"
+    }
+  }
+  
```

Comment:
file is complaining about no new line. It might be because there's white spaces on line 16? same issue on some other files